draft-ietf-rtcweb-jsep.xml

```diff
@@ -4076,6 +4076,65 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             process, as a remote offer or answer, unmodified SDP coming
             from any other implementation of this specification.</t>
         </section>
+        <section title="Processing RTP packets"
+                 anchor="sec.rtp.demux">
```

Comment:
I don't think we should demux *between* RtpReceivers using RID.  RIDs are scoped to MIDs, and so you would need to be careful to make all your RIDs unique across and entire transport for that to work.  It's much more safe to just use MIDs for demuxing between RtpReceivers, and then only use RIDs within an RtpReceiver to known which RTP streams repair which other RTP streams.

You can demux before receiving an answer using MIDs first (to choose between the RtpReceivers) and RIDs second (to know which RTP stream a repair stream repairs).

I added a paragraph at the end to try and make this more clear.